.circleci/config.yml

```diff
@@ -531,6 +501,15 @@ jobs:
             - /tmp/go-cache
       - store_artifacts:
           path: bin/linux
+      - setup_remote_docker:
+          version: 19.03.13
+          docker_layer_caching: true
+      - run:
+          name: Build the candidate docker image
```

Comment:
Do we want to skip this step when building from a fork? Does it add any significant runtime to the build stage?

Reply:
Alt: since we're already publishing the `influx` and `influxd` binaries as artifacts from this stage, could we put the Docker-build logic in the job that runs the monitor-ci tests?

Reply:
Since the binary is already built it goes pretty fast - takes about a minute. That said, every little bit helps, so it would make sense to skip the step for forks. That is, when we start using the `monitor-ci` pipeline, because right now even forks would need it since they'll use it during the `e2e` job.

Initially I had actually put the image building stuff in the `e2e` job, but then I got to thinking you'd end up rebuilding the docker image any time you needed to re-run the `e2e` job - which is more often than the `build` job. I think this will continue to be the case with the `monitor-ci` pipeline...e2e tests are finicky. So it would have the potential to add more time than needed in the case of re-running the e2e tests.

After that I actually tried building the docker image as its own separate job. That works fine but there's a bit of overhead from spinning up the environment to build it in. So I ended up here...the binary is right there, so just build it into the image.